src/rpc/RPCHelpers.cpp

```diff
@@ -81,6 +81,7 @@
 
 #include <algorithm>
 #include <array>
+#include <cassert>
```

Comment:
fyi, this is used line [375](https://github.com/XRPLF/clio/pull/1712/files#diff-63a13cef09d0b504ba9246cab2dfade8752f6cca5640b218f6885f3881d385cfL375)